### ISSUE TITLE
Archive some more repositories

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/standard#529]
I archived all the `ocds_upgrade_*` repos and `ocds-milestones-update`. I didn't archive `ocds_upgrade_1_1_patches` because #229 is still open.

Admins can see audit log here: https://github.com/organizations/open-contracting-archive/settings/audit-log

I propose the archival of (assuming none of these are used anymore, or linked from standard.open-contracting.org and other current materials):

* [ ] https://github.com/open-contracting/standard-legacy-staticsites
* [x] https://github.com/open-contracting/jsonmerge
* [x] https://github.com/open-contracting/ocds-conformance
* [x] https://github.com/open-contracting/opendatacomparison
* [x] https://github.com/open-contracting/use-case-demo
* [x] https://github.com/open-contracting/pycon-sprint-2014

https://github.com/open-contracting/open-contracting.github.io I suppose needs to stay around in order to have the `meta-refresh` redirect to standard.open-contracting.org

**Comment by timgdavies** [comment]
Thanks for the tidy up. 

I've archived ```ocds_upgrade_1_1_patches``` as #229 is likely to lead to new work if we do progress it. 

I've checked off above those that I think can be archived safely. Will flag to the rest of the team to confirm, but if no objections by end of day, I think safe to archive those too. 

I believe we should keep https://github.com/open-contracting/standard-legacy-staticsites (checking with @bjwebb) as these are part of the deployed documentation.

**Comment by jpmckinney** [comment]
Sounds good. When was the switch made from JSON-Merge to JSON-Merge-Patch? Just wondering about https://github.com/open-contracting/ocds-merge

**Comment by timgdavies** [comment]
For extensions we made the switch from [JSON Patch](http://jsonpatch.com/) to [JSON Merge Patch](https://tools.ietf.org/html/rfc7386) in [September 2016](https://github.com/open-contracting/standard/issues/335#issuecomment-246769257). 

We maintain a [python implementation of a JSON Merge Patch library](https://github.com/open-contracting/json-merge-patch) (written by @kindly). This could potentially move to the OpenDataServices repo, as I suspect we'll maintain it whatever, though it was originally written on OCP time. 

[jsonmerge](https://github.com/open-contracting/jsonmerge) and [ocds-merge](https://github.com/open-contracting/ocds-merge) are concerned with the actual [merging of OCDS releases](http://standard.open-contracting.org/latest/en/schema/merging/), rather than extension management. 

I believe [jsonmerge](https://github.com/open-contracting/jsonmerge)is now superseded by [ocds-merge](https://github.com/open-contracting/ocds-merge) and so jsonmerge should be archived.

**Comment by jpmckinney** [comment]
Thanks! Added jsonmerge to list at top of issue.

**Comment by jpmckinney** [comment]
Seems fine to me to move `json-merge-patch` to `OpenDataServices`. Where do these belong?

* https://github.com/OpenDataServices/OCDSExtensionCreator
* https://github.com/OpenDataServices/ocds-faker

**Comment by timgdavies** [comment]
https://github.com/OpenDataServices/OCDSExtensionCreator is an early draft of https://github.com/open-contracting/extension_creator - I'll check if we can get this removed.

https://github.com/OpenDataServices/ocds-faker is currently used for generating test data. We could move it to open-contracting to support it here, but probably only once it is more documented.

**Comment by Bjwebb** [comment]
> I believe we should keep https://github.com/open-contracting/standard-legacy-staticsites (checking with @Bjwebb) as these are part of the deployed documentation.

That's correct.

**Comment by jpmckinney** [comment]
I archived the checked repos in issue description.
